apps/test/unit/lib/kits/maker/ui/SetupGuideTest.js

```diff
@@ -0,0 +1,68 @@
+/** @file Test SetupGuide component */
+import React from 'react';
+import {expect} from '../../../../../util/deprecatedChai';
+import {shallow} from 'enzyme';
+import SetupGuide from '@cdo/apps/lib/kits/maker/ui/SetupGuide';
+import applabI18n from '@cdo/applab/locale';
+import experiments from '@cdo/apps/util/experiments';
+
+describe('MakerSetupGuide', () => {
+  describe('Microbit experiment is not enabled', () => {
+    it('renders circuit playground description', () => {
+      let [description, markdown] = getGuideContent(
+        '#circuit-playground-description'
+      );
+      expect(description.prop('title')).to.contain(
+        applabI18n.makerSetupCircuitPlaygroundTitle()
```

Comment:
These should be stubbed to ensure that these functions are actually being called... but you pull this from a const in the beginning of the `SetupGuide.jsx` so it gets called once before your tests are ever made. This is tricky! You might either have to ensure that the i18n content is determined at the render time of the page instead of the script load, or just not stub this to test it.

Reply:
1. Constant turned into function.
2. i18n function is stubbed

Reply:
1. Constant turned into function.
2. i18n function is stubbed